robot_smach_states/src/robot_smach_states/manipulation/manipulation.py

```diff
@@ -262,80 +277,88 @@ def __init__(self, side, x, y, z, roll=0, pitch=0, yaw=0, time_out=20, pre_grasp
         @param point_designator resolves to a PointStamped the gripper should move to"""
-        smach.State.__init__(self, outcomes=['succeeded','failed'])
-        self.robot                = robot
-        self.side                 = side
-        self.point_designator     = point_designator
-        self.time_out             = time_out
-        self.pre_grasp            = pre_grasp
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+        self.robot = robot
+        self.side = side
+        self.point_designator = point_designator
+        self.time_out = time_out
+        self.pre_grasp = pre_grasp
         self.first_joint_pos_only = first_joint_pos_only
 
     def execute(self, userdata=None):
         # ToDo: check point_designator?
         goal = self.point_designator.resolve()
         if not goal:
-            rospy.loginfo("point_designator {0} cannot be resolved: {1}".format(self.point_designator))
+            rospy.loginfo("point_designator {0} cannot be resolved.".format(self.point_designator))
             return 'failed'
 
         rospy.loginfo("ArmToQueryPoint: goal = {0}".format(goal))
 
         # Note: answers are typically in "map"frame, check whether this works out
         rospy.logwarn("Transforming to base_link frame for amigo_arm_navigation")
-        goal_bl = transformations.tf_transform(goal.point, goal.header.frame_id, "/amigo/base_link", tf_listener=self.robot.tf_listener)
-        if goal_bl == None:
+        goal_bl = transformations.tf_transform(goal.point, goal.header.frame_id, "/amigo/base_link",
+                                               tf_listener=self.robot.tf_listener)
+        if goal_bl is None:
             return 'failed'
 
         if self.side.send_goal(goal_bl.x, goal_bl.y, goal_bl.z, 0, 0, 0,
-            frame_id="/amigo/base_link",
-            timeout=self.time_out,
-            pre_grasp=self.pre_grasp,
-            first_joint_pos_only=self.first_joint_pos_only):
+                               frame_id="/amigo/base_link",
+                               timeout=self.time_out,
+                               pre_grasp=self.pre_grasp,
+                               first_joint_pos_only=self.first_joint_pos_only):
             return 'succeeded'
         else:
             rospy.logwarn("Goal unreachable: {0}".format(goal_bl).replace("\n", " "))
             return 'failed'
 
+
 class TorsoToUserPos(smach.State):
     def __init__(self, robot, torso_pos, time_out=0.0):
-        smach.State.__init__(self, outcomes=['succeeded','failed'])
+        """
+        State to set the pose of the torso
+        :param robot: robot to execute state with
+        :param torso_pos: float desired position of the torso
+        :param time_out: float amount of time the procedure may take
+        """
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
         self.robot = robot
         self.torso_pos = torso_pos
         self.time_out = time_out
```

Comment:
The ArmToQueryPoint class may be deprecated. It is not used in any of the challenges. Should this function be removed?